src/dodal/common/scanspecs.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Optional
+
+import numpy as np
+from scanspec.core import Frames
+from scanspec.specs import DURATION
```

Comment:
Should: Having this file called `scanspecs` and referencing `scanspec` is going to get confusing. Can we either move this function into scanspec if it makes sense to do so or call this something like `scanspec_utils`?

Reply:
Will make a PR into Scanspec

Reply:
Merged https://github.com/dls-controls/scanspec/pull/93